config/environments/development.rb

```diff
@@ -2,7 +2,7 @@
 
 Rails.application.configure do
   # Verifies that versions and hashed value of the package contents in the project's package.json
-  config.webpacker.check_yarn_integrity = true
+  config.webpacker.check_yarn_integrity = false
```

Comment:
so, what did we need this off?

Reply:
It's failing in docker and causing it not to start:

```
web_1  | => Booting Puma
web_1  | => Rails 5.2.3 application starting in development
web_1  | => Run `rails server -h` for more startup options
web_1  |
web_1  |
web_1  | ========================================
web_1  |   Your Yarn packages are out of date!
web_1  |   Please run `yarn install` to update.
web_1  | ========================================
web_1  |
web_1  |
web_1  | To disable this check, please add `config.webpacker.check_yarn_integrity = false`
web_1  | to your Rails development config file (config/environments/development.rb).
web_1  |
web_1  |
web_1  | yarn check v1.15.2
web_1  | warning Integrity check: Flags don't match
web_1  | error Integrity check failed
web_1  | error Found 1 errors.
...
web_1  | Exiting
```
I think it's because of this:

```
info fsevents@1.2.4: The platform "linux" is incompatible with this module.
info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.
```

Reply:
Ah - worked it out - it was the `NODE_ENV=production` setting in the `docker_web` command. It was installing dev dependencies and then getting confused that they weren't there. So we can put this back :)